common-npm-packages/azure-arm-rest-v2/azure-arm-storage.ts

```diff
@@ -212,6 +212,45 @@ export class StorageAccounts {
         return storageAccounts[index];
     }
 
+    public async getStorageAccountProperties(resourceGroupName: string, storageAccountName: string): Promise<Model.StorageAccount | undefined> {
+        if (resourceGroupName === null || resourceGroupName === undefined || typeof resourceGroupName.valueOf() !== 'string') {
+            throw new Error(tl.loc("ResourceGroupCannotBeNull"));
+        }
+
+        if (storageAccountName === null || storageAccountName === undefined || typeof storageAccountName.valueOf() !== 'string') {
+            throw new Error(tl.loc("StorageAccountCannotBeNull"));
+        }
+
+        let response = await this.sendRequest(
+            "GET",
+            "//subscriptions/{subscriptionId}/resourcegroups/{resourceGroupName}/providers/Microsoft.Storage/storageAccounts/{storageAccountName}",
+            {
+                '{resourceGroupName}': resourceGroupName,
+                '{storageAccountName}': storageAccountName
+            }
```

Comment:
Is there a reason why don't we pass `{subscriptionId}` in this call?

Reply:
Yes, it passes automatically using [AzureServiceClient](https://github.com/microsoft/azure-pipelines-tasks/blob/master/common-npm-packages/azure-arm-rest-v2/AzureServiceClient.ts#L17), added description for the method, thank!